tests/ctst/steps/replication.ts

```diff
@@ -89,8 +89,11 @@ When('the job to replicate existing objects with status {string} is executed',
         await createAndRunPod(this, podManifest);
     });
 
-Then('the object should eventually {string} replicated', { timeout: 360_000 },
-    async function (this: Zenko, replicate: 'be' | 'fail to be') {
+Then('the object replication should eventually {string}', { timeout: 360_000 },
```

Comment:
nit: removing the `eventually` makes it easier to read, esp. in the "never happen" case
```suggestion
Then('the object replication should {string}', { timeout: 360_000 },
```